backend/models.py

```diff
@@ -104,14 +104,18 @@ class Schema(Base):
 
 class Entity(Base):
     __tablename__ = 'entities'
-
+    
     id = Column(Integer, primary_key=True, index=True)
-    name = Column(String(128), nullable=False)
+    slug = Column(String(128), nullable=False)
```

Comment:
Actually, I think it makes sense to have both: `name` and `slug` :thinking:

Reply:
What `name` will be used for in this case? I mean, we can use `slug` both as `id` replacement and as something like title for entity to display it on frontend

Reply:
> validator for slug to make sure it's indeed URL-friendly

This one is actually needed

> `name` is the human-friendly identifier/name/title.

Well, on frontend we can transform slug and replace `-` and `_` with spaces for example😏

Reply:
Yes, that is an option.

Just thinking out loud here: For users it might be more intuitive, if it's the other way. I'm thinking of this workflow:

* User fills out an input form (to create an entity).
* While typing in the field for `name`, the field for `slug` gets automatically filled.
* When editing an entity, users can change `name` and `slug` independently.

This way we would set some decent defaults and the entity URL (with slug) would not change, if the name actually needs to change.

In addition, we could make  `(schema_id, slug)` unique and allow names to be non-unique :thinking:

Reply:
Alright, but now we get back to one of my previous questions

> allow names to be non-unique

Since `name` works as, basically, predefined attribute, users should be able to make it either unique or non-unique on specific schema. Should we add then a flag to `Schema` that would show uniqueness of `name`?